bin/configs/typescript-fetch-utility-types-without-runtime.yaml

```diff
@@ -0,0 +1,10 @@
+generatorName: typescript-fetch
```

Comment:
thanks for adding the sample configs here, but after looking at the updated unit test, which covers the relevant code changes, i would argue the sample configs bin/configs/typescript-fetch-utility-types-without-runtime.yaml and bin/configs/typescript-fetch-utility-types.yaml and the corresponding samples should be removed, since:

- the diff to other samples is minimal
- the functionality is already covered in the unit test
- there is no big change in the template